src/lib/low_latency_schedule.c

```diff
@@ -520,24 +520,24 @@ static struct work_queue *work_new_queue(struct work_queue_timesource *ts)
 	return queue;
 }
 
-void init_system_workq(struct work_queue_timesource *ts)
+void init_system_llq(struct work_queue_timesource *ts)
 {
-	struct work_queue **queue = arch_work_queue_get();
+	struct ll_queue **queue = arch_ll_queue_get();
 
 	*queue = work_new_queue(ts);
 
 	if (cpu_get_id() == PLATFORM_MASTER_CORE_ID) {
-		work_shared_ctx = rzalloc(RZONE_SYS | RZONE_FLAG_UNCACHED,
+		ll_shared_ctx = rzalloc(RZONE_SYS | RZONE_FLAG_UNCACHED,
 					  SOF_MEM_CAPS_RAM,
-					  sizeof(*work_shared_ctx));
-		atomic_init(&work_shared_ctx->total_num_work, 0);
-		atomic_init(&work_shared_ctx->timer_clients, 0);
+					  sizeof(*ll_shared_ctx));
+		atomic_init(&ll_shared_ctx->total_num_work, 0);
+		atomic_init(&ll_shared_ctx->timer_clients, 0);
 	}
 }
 
-void free_system_workq(void)
+void free_system_llq(void)
```

Comment:
So this file would export a structure of scheduler API function pointers that the core scheduling API would use if this scheduler was selected by the client.

Reply:
Done